src/components/SupportingPartners/index.js

```diff
@@ -0,0 +1,72 @@
+import { A } from "@commons-ui/core";
+import { Grid, Typography } from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
+import Image from "next/image";
+import PropTypes from "prop-types";
+import React from "react";
+
+import Section from "@/pesayetu/components/Section";
+
+const useStyles = makeStyles(({ palette, typography }) => ({
+  root: {
+    paddingTop: typography.pxToRem(56.69),
+    paddingBottom: typography.pxToRem(80),
+    background: palette.grey.light,
+  },
+  section: {},
+  title: {
+    marginBottom: typography.pxToRem(49.38),
+    textTransform: "uppercase",
+  },
+  logoContainer: {},
+  link: {},
+  logo: {
+    margin: "0",
+  },
+}));
+function SupportingPartners({ title, items, ...props }) {
+  const classes = useStyles(props);
+
+  return (
+    <div className={classes.root}>
+      <Section className={classes.section}>
+        <Typography variant="h4" className={classes.title}>
+          {title}
+        </Typography>
+        <Grid container spacing={2} className={classes.logoContainer}>
+          {items?.map(({ link, logo, name }) => (
```

Comment:
Lets check before returning that we do have some `items`. Otherwise, there is no point in showing empty section.